plugin/db/mysql/mysql.go

```diff
@@ -44,6 +46,20 @@ func init() {
 	db.Register(db.TiDB, newDriver)
 }
 
+// BinlogInfo is the binlog coordination for MySQL.
+type BinlogInfo struct {
+	Filename string `json:"file_name"`
+	Position int64  `json:"position"`
+}
+
+// This is encoded in JSON and stored in the backup table, representing PITR related info.
+type backupPayload struct {
+	BinlogInfo BinlogInfo `json:"binlog_info"`
+	// Imprecise UNIX timestamp to the second which is the rough time when this backup is taken.
+	// Mainly for UI purpose.
+	ImpreciseTs int64 `json:"imprecise_ts"`
```

Comment:
I would just call it "ts", imprecise is an implementation detail. What if we later find a method that we can get an accurate ts??